modules/generator.rb

```diff
@@ -0,0 +1,77 @@
+class Generator
+  attr_reader :readers, :authors, :books, :orders
+
+  def initialize
+    @readers = []
+    @authors = []
+    @books = []
+    @orders = []
+  end
+
+  def init_library
+    init_authors
+    init_books
+    init_readers
+    init_orders
+    save_obj_to_file
+  end
+
+  def random_add(obj)
```

Comment:
Will be better, to rename `obj` to `library`, because you don't have any another object with the same behaviour